modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/helper/DDMFormRendererHelper.java

```diff
@@ -130,7 +133,12 @@ private DDMFormValues _getDDMFormValues(DDMForm ddmForm) {
 					_configurationModel, ddmForm, _getLocale(),
 					_locationVariableResolver);
 
-		return configurationModelToDDMFormValuesConverter.getDDMFormValues();
+		DDMFormValues ddmFormValues =
+			configurationModelToDDMFormValuesConverter.getDDMFormValues();
+
+		_validateDDMFormValuesWithOverrideConfiguration(ddmFormValues);
```

Comment:
This validation maybe shouldn't be done in the `ConfigurationModelToDDMFormValuesConverter` class? What do you think @ethib137 ?

Reply:
Hi @matuzalemsteles, for what I've noticed, when reproducing the bug, the code doesn't stop at `ConfigurationModelToDDMFormValuesConverter`, furthermore, the mentioned class doesn't have the required elements for the validation.

Reply:
Done!